examples/exports_memory.rs

```diff
@@ -37,7 +37,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     // the default provided by Wasmer.
     // You can use `Store::default()` for that.
     let mut store = Store::new(Cranelift::default());
-    let mut env = FunctionEnv::new(&mut store, ());
+    let _env = FunctionEnv::new(&mut store, ());
```

Comment:
If it's not used, we can just remove this line :)

Reply:
done